content/en/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -195,9 +194,9 @@ The following file is an Ingress resource that sends traffic to your Service via
 
 1. Verify that the Ingress contorller is directing traffic:
 
```

Comment:
nit: line 195, correct spelling of controller
nit: should this be item number 5? Probably an extra line needs to be taken away or added.